main.py

```
@@ -11210,6 +11014,254 @@ async def count(interaction: discord.Interaction) -> None:
+
+@bot.command(name="emojis")
+@is_bot_owner()
+async def owner_emojis(ctx: commands.Context) -> None:
+    global emojis
+    emojis = {emoji.name: str(emoji) for emoji in await bot.fetch_application_emojis()}
+    try:
+        async with await anyio.open_file("config/emojis_cache.json", "w", encoding="utf-8") as f:
+            await f.write(json.dumps(emojis))
+        await ctx.reply("emojis refreshed!")
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)